Board.elm

```diff
@@ -5,7 +5,10 @@ module Board exposing (init, view, update, subscriptions)
 
 import Html exposing (Html)
 import Html.App
+
+
 -- import Html.Attributes exposing (..)
+
```

Comment:
I've completely deleted the `Html.Attributes` line.  My philosophy is generally that, since we have a very capable source-code system, it's easy to go back and find what we "were thinking about."

Hence, just completely delete the commented out line.